src/main/java/org/broadinstitute/consent/http/models/Presentation.java

```diff
@@ -0,0 +1,4 @@
+package org.broadinstitute.consent.http.models;
+
+public record Presentation(String title, String link, String date, String authors, String citation) {
```

Comment:
Should `authors` be an array? Should date be a `Date` object, or link be `URI`? Not sure if whatever parsing library we're using supports those objects

Reply:
I'm hesitant to model these any more granularly than generic strings. The current mocks are super simple. For example, this is a presentation:
<img width="1000" alt="Screenshot 2023-05-02 at 12 15 48 PM" src="https://user-images.githubusercontent.com/116679/235724577-709cd79b-e1e6-4deb-912b-dcc655c78127.png">

And this is a publication:
<img width="979" alt="Screenshot 2023-05-02 at 12 16 33 PM" src="https://user-images.githubusercontent.com/116679/235724709-d481a9d5-b2ce-4a70-a2c3-c4e012820d8a.png">

However, looking at this again, I think citation needs to be a boolean, not a string. I think the earlier versions were not booleans but looking at the mocks again, they are.